student.rb

```diff
@@ -7,4 +7,9 @@ def initialize(age, classroom, name = 'UnKnown', parent_permission: true)
   def play_hooky
     '¯(ツ)/¯'
   end
+
+  def enroll(classroom)
+    @classroom = classroom
+    classroom.students << self unless classroom.students.include?(self)
+  end
```

Comment:
- [ ] Hey there! 👋 Awesome job in implementing the method that establishes the many-to-one association between a student and a classroom! 🌟 However, there's a slight inconsistency: in the Classroom method, you referred to student.classroom, but in the Classroom class, you've created an enrol method. To maintain consistency and clarity, consider adjusting the method name in the Student class to classroom to align with its definition. 🔄 This small tweak enhances readability and ensures a smooth understanding of the association between students and classrooms. remember to include getter for classroom as well. 💡💻 Keep up the great work! 🚀

```suggestion
attr_reader :classroom
  def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```